app/presenters/api_docs/api_schema.rb

```diff
@@ -57,6 +57,7 @@ def type_description
           desc << " (limited to #{attributes.max_length} characters)"
         elsif type == 'array' && attributes.max_items.present?
           desc << " (limited to #{attributes.max_items} elements)"
+          desc << " of stings (limited to #{attributes.items.max_length} characters)" if limited_string_within_array
```

Comment:
```suggestion
          desc << " of strings (limited to #{attributes.items.max_length} characters)" if limited_string_within_array
```

Imagining an array of Stings 😂

Reply:
🤦‍♀️